tests/jballerina-unit-test/src/test/resources/test-src/variable/shadowing/shadowing.bal

```diff
@@ -184,3 +187,19 @@ function testTypeNameAsVariable5() {
     var t = typeof name;
     panic error(ASSERTION_ERROR_REASON, message = "expected 'string', found '" + t.toString() + "'");
 }
+
+function testGeneratedNames() {
+    assertEquality(serviceEPAvailable, false);
+    assertEquality(LOCK_STORE, "lockStore");
```

Comment:
you can use `test:assertFalse` and `test:assertEquals` from the `ballerina/test` package